test/SILGen/opaque_values_silgen.swift

```diff
@@ -1,5 +1,7 @@
 // RUN: %target-swift-frontend -enable-sil-opaque-values -emit-sorted-sil -Xllvm -new-mangling-for-tests -Xllvm -sil-full-demangle -emit-silgen %s | %FileCheck %s
 
+// REQUIRES: no_resilient_stdlib
```

Comment:
Please use UNSUPPORTED instead of adding a "no_" feature.

(Actually, @gparker42, did the new boolean expression improvements allow for `!`?)